prisma/schema.prisma

```diff
@@ -115,16 +119,100 @@ model Area {
   question             String?
   assessors_guide      String?
   stageId              Int
-  stage                Stage      @relation(fields: [stageId], references: [id])
+  stage                Stage            @relation(fields: [stageId], references: [id])
   artefacts            Artefact[]
+  AssessmentArea       AssessmentArea[]
+  Score                Score[]
 
   @@unique([area_id, area_name, stageId])
 }
 
-
 model Stage {
   stageNumber Int
-  id          Int    @id @default(autoincrement())
-  name        String @unique
+  id          Int     @id @default(autoincrement())
+  name        String  @unique
   areas       Area[]
+  Score       Score[]
+}
+
+// New Models
+
+model Project {
```

Comment:
Add a status? Should be trivial. Although I don't want to necessarily have that logic too since it's secondary. Let's settle with a status "archived" rather than an "in progress" or so.